cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/PreferencesHttpHandlerTest.java

```diff
@@ -48,7 +48,7 @@ public static void init() {
 
   private void addApplication(String namespace, Application app) {
     ApplicationSpecification appSpec = Specifications.from(app);
-    store.addApplication(new NamespaceId(namespace).app(appSpec.getName()).toId(), appSpec);
+    store.addApplication(new NamespaceId(namespace).app(appSpec.getName()), appSpec);
```

Comment:
when you don't already have a ``NamespaceId``, its better to create ``ApplicationId`` directly, so it is a little more concise - ``new ApplicationId(namespace, appSpec.getName)``